Modules/Core/ImageFunction/include/itkBSplineInterpolateImageFunction.h

```diff
@@ -415,9 +415,9 @@ class ITK_TEMPLATE_EXPORT BSplineInterpolateImageFunction : public InterpolateIm
   bool m_UseImageDirection{};
 
   ThreadIdType                          m_NumberOfWorkUnits{};
-  std::unique_ptr<vnl_matrix<long>[]>   m_ThreadedEvaluateIndex {};
-  std::unique_ptr<vnl_matrix<double>[]> m_ThreadedWeights {};
-  std::unique_ptr<vnl_matrix<double>[]> m_ThreadedWeightsDerivative {};
+  std::unique_ptr<vnl_matrix<long>[]>   m_ThreadedEvaluateIndex ;
```

Comment:
Remove space before semicolon. linter [complains](https://github.com/InsightSoftwareConsortium/ITK/actions/runs/3968371684/jobs/6801455642)

Reply:
These spaces were actually introduced automatically by Clang-Format (with by PR #3851), when adding the `{}`! I think it's a Clang-Format bug!

Reply:
Just force pushed a slightly modified version of the regexp to cope with this, linter is happy now!